rancher/latest/en/kubernetes/ingress/index.md

```diff
@@ -0,0 +1,291 @@
+
+The application will:
+
+* Listen to Kubernetes server events
+* Deploy a load balancer and program it with the routing rules defined in Ingress
+* Configure the Ingress `Address` field with the Public Endpoint of your Load Balancer
+
+### Creating Ingress resources in Rancher
+
+#### Setting up an Example Nginx Service
+
+Before setting up any ingress, services will need to be created in Kubernetes. We'll start by adding a service and replication controller into our Kubernetes environment. 
+
+Here's a simple nginx service to add into Kubernetes.
+
+Example `nginx-service.yml`:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginx-service
+  labels:
+    k8s-app: nginx-service
+spec:
+  ports:
+  - port: 90
+    targetPort: 80
+    protocol: TCP
+    name: http
+  selector:
+    k8s-app: nginx-service
+---
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: nginx-service
+spec:
+  replicas: 1
+  selector:
+    k8s-app: nginx-service
+  template:
+    metadata:
+      labels:
+        k8s-app: nginx-service
+    spec:
+      terminationGracePeriodSeconds: 60
+      containers:
+      - name: nginx-service
+        image: nginx:latest
+        ports:
+        - containerPort: 80
+```
+
+Using `kubectl`, let's launch our nginx service into Rancher. Remember, you can either [configure `kubectl` for your local machine]({{site.baseurl}}/rancher/{{page.version}}/{{page.lang}}/kubernetes/#kubectl) or you can use the shell in the UI under **Kubernetes** -> **kubectl**.
+
+```bash
+$ kubectl create -f nginx-service.yml
+service "nginx-service" created
+replicationcontroller "nginx-service" created
+```
+
+#### Setting up a Simple Ingress Resource
+
+You can set up a simple ingress resource for the single service. 
+
+Example `simple-ingress.yml`
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: simplelb
+spec:
+  backend:
+    serviceName: nginx-service
+    servicePort: 90
+```
+
+Let's create the ingress using `kubectl`. After you create the ingress, a load balancer will be created and visible in the **kubernetes-ingress-lbs** stack within the **Kubernetes** -> **System** tab. From `kubectl`, you can get the ingress created, but the UI will only show the load balancer, which has translated the requests in the ingress to a Rancher load balancer.  
+
+```bash
+$ kubectl create -f simple-ingress.yml
+ingress "simplelb" created
+$ kubectl get ingress
+NAME          RULE      BACKEND            ADDRESS          AGE
+simplelb      -         nginx-service:80   1.2.3.4          5m
+```
+
+The `address` in the ingress will be the public endpoint of where the load balancer service was launched. If the load balancer is moved to a different host and gets a different public endpoint, the ingress `address` will be updated. 
+
+#### Setting up Multiple Services
+
+If there are multiple services in the ingress, then you could add host/path based routing to your ingress to balance between the services. 
```

Comment:
I would say that if you want your ingress to act as an entry point for more than one service, then you would have to configure it with the host name routing rules.